versions.tf

```
@@ -0,0 +1,19 @@
+terraform {
+  required_version = ">= 1.5.5"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.12.0"
+    }
+    archive = {
+      version = ">= 2.4.0"
+    }

```

É interessante manter garantias de API, já que esses providers usam versionamento semântico.

```suggestion
    aws = {
      source  = "hashicorp/aws"
      version = "~> 5.12"
    }
    archive = {
      version = "~> 2.4"
    }

```